dashbuilder-backend/dashbuilder-dataset-cdi/pom.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2014 JBoss Inc
```

Comment:
The header should not contain references to JBoss Inc (it does not exist for some time now), see https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/ide-configuration/LICENSE-ASL-2.0-HEADER.txt